src/sentry/tagstore/migrations/0001_initial.py

```diff
@@ -0,0 +1,235 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
```

Comment:
I would argue that we should just set this to True for ourselves even though it's not technically dangerous, just applying this to our existing cluster doesn't need to happen. We'll apply manually to shards.